huaweicloud/config.go

```diff
@@ -444,6 +471,37 @@ func (c *Config) newServiceClientByName(client *golangsdk.ProviderClient, catalo
 	return sc, nil
 }
 
+// loadUserProjects will query the region-projectId pair and store it into RegionProjectIDMap
+func (c *Config) loadUserProjects(client *golangsdk.ProviderClient, region string) error {
+
+	domainID := client.DomainID
+	opts := projects.ListOpts{
+		DomainID: domainID,
+		Name:     region,
+	}
+	sc := new(golangsdk.ServiceClient)
+	sc.Endpoint = fmt.Sprintf("https://iam.%s/v3/", c.Cloud)
```

Comment:
suggest to set Endpoint by `c.IdentityEndpoint`

Reply:
Yeah, and I have modified about this.